### ISSUE TITLE
[BUG] Undefined variable: organisers

### ISSUE THREAD

**Issue by pendor** [Attendize/Attendize#871]
New install gives error "Undefined variable: organisers" after initial registration.

![image](https://user-images.githubusercontent.com/209193/102632718-72c06300-411d-11eb-92ed-ee4d2f53e36f.png)

app/Http/Middleware/SetViewVariables.php should be altered to share the organisers variable outside of the check for "if (Utils::installed())".

IE:
`public function handle($request, Closure $next)
    {
        /*
         * Share the organizers across all views
         */
        View::share('organisers', Organiser::scope()->get());

        if (Utils::installed()) {

            /*
             * Set up JS across all views
             */
            JavaScript::put([
// ...
`

This change allows the select_organiser page to load as well as the dashboard after it.  Both would show the same error otherwise.  This error exists in 7d5da3424151b0c92ef3fbfb85785bd1d6ecc16c

**Comment by ranfysvalle02** [comment]
@pendor  - this happened to me, and here is how I solved it.

[attendly_url]/install, and try to re-install. Attempting to re-install, the system will pick up it was already configured when I attempts to connect to the database, and the errors stop. 

I think a simple fix is to always force endpoint /install , unless the configuration is already set up. 

This is how I got around a similar situation - unfortunately I can't guarantee it will help in your case but just my 2 cents.
